CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## [MAJOR.MINOR.PATCH] - YYYY-MM-DD
 - Add Cassandra Kind
 - Add Grafana Kind
+- Recreate Kafka ACL if modified. 
```

Comment:
Is this actually true that we need to ask users to delete ACL? Since we delete the old ACL before creating a new one.

Reply:
Old ACLs doesn't have id. They are found in the `List()` by topic name, username and permission. If you modify the manifest you can't find them, hence delete it. That is the thing, yes (